### PR TITLE
Frontend/feature/homepage

### PR DESCRIPTION
**Pull request by bekirtatar**
The left sidebar is implemented with tab functionality. The middle part of the page is created with some static values.

### CODE REVIEW

File: source/frontend/package.json [hunk 1]
```diff
@@ -14,11 +14,12 @@
     "@mui/icons-material": "^5.1.0",
     "@mui/material": "^5.0.6",
     "@mui/styles": "^5.1.0",
+    "@parcel/packager-js": "^2.0.1",
     "@reduxjs/toolkit": "^1.6.2",
     "@types/react-redux": "^7.1.20",
-    "@types/redux-saga": "^0.10.5",
     "all": "^0.0.0",
     "axios": "^0.24.0",
+    "parcel": "^2.0.1",
```
Comment (emindeniz99) [inline source/frontend/package.json:22]: It is devDependencies

File: source/frontend/package.json [hunk 2]
```diff
@@ -14,11 +14,12 @@
     "@mui/icons-material": "^5.1.0",
     "@mui/material": "^5.0.6",
     "@mui/styles": "^5.1.0",
+    "@parcel/packager-js": "^2.0.1",
```
Comment (emindeniz99) [inline source/frontend/package.json:17]: What is this?

File: source/frontend/package.json (hunk 1 above)
Comment (emindeniz99) [inline source/frontend/package.json:22]: It is devDependencies

**Review by emindeniz99 — APPROVED** [review]

File: source/frontend/package.json (hunk 2 above)
Comment (bekirtatar) [reply source/frontend/package.json:17]: It is used to test a build bug.

**Pull request merged**